### PR TITLE
Change order model

### PR DESCRIPTION
**Pull request by Platton96**

### CODE REVIEW

**Comment by Platton96** [comment]
fix #153

File: ecommers/lib/ui/pages/more_page.dart [hunk 1]
```diff
@@ -51,53 +54,54 @@ class MorePage extends StatelessWidget {
...
+        scrollDirection: Axis.vertical,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: Insets.x6),
+              child: Text(
+                I18n.of(context).morePage,
+                style: Theme.of(context).textTheme.headline6,
```
Comment (fryette) [inline ecommers/lib/ui/pages/more_page.dart]: use text theme

File: ecommers/lib/ui/pages/more_page.dart [hunk 2]
```diff
@@ -51,53 +54,54 @@ class MorePage extends StatelessWidget {
...
+      child: SingleChildScrollView(
+        scrollDirection: Axis.vertical,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: Insets.x6),
+              child: Text(
+                I18n.of(context).morePage,
```
Comment (fryette) [inline ecommers/lib/ui/pages/more_page.dart]: remove and use localization

File: ecommers/lib/core/repositories/repository_base.dart [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  final String filterFieldForItem;
+  final String repositoryKey;
+  
+  RepositoryBase({
+    this.filterFieldForItem,
+    this.repositoryKey,
+  });
+
+  Future edit(T item) async {
+    await cacheDatabase.updateByEqualsFilter(
```
Comment (fryette) [inline ecommers/lib/core/repositories/repository_base.dart]: can be without async and just return(please check whole file)

**Review by fryette — CHANGES_REQUESTED** [review]

File: ecommers/lib/ui/pages/more_page.dart (hunk 1 above)
Comment (Platton96) [reply ecommers/lib/ui/pages/more_page.dart]: changed

File: ecommers/lib/ui/pages/more_page.dart (hunk 2 above)
Comment (Platton96) [reply ecommers/lib/ui/pages/more_page.dart]: removed

File: ecommers/lib/core/repositories/repository_base.dart (hunk 3 above)
Comment (Platton96) [reply ecommers/lib/core/repositories/repository_base.dart]: fuxed

File: ecommers/lib/ui/pages/more_page.dart [hunk 4]
```diff
@@ -51,53 +53,54 @@ class MorePage extends StatelessWidget {
...
-      child: CupertinoButton(
-        onPressed: logOutPressHandler,
-        child: Text(
-          localization.logOut,
-          style: textTheme
-              .headline5
-              .apply(color: BrandingColors.primary),
-        ), //TODO use proovider
-      ),
+    return Consumer<MoreProviderModel>(
```
Comment (amarchenko25) [inline ecommers/lib/ui/pages/more_page.dart]: do u really need consumer here? Because u use it just for getting provider object.

Comment (Platton96) [reply ecommers/lib/ui/pages/more_page.dart]: remove consumer

**Review by amarchenko25 — APPROVED** [review]

**Pull request merged**